Userland/Services/SystemServer/main.cpp

```diff
@@ -490,6 +494,8 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
         TRY(determine_system_mode());
     }
 
+    dbgln("{}", g_system_mode);
```

Comment:
Isn't the double logging unnecessary, especially this line where we log no other context?